aptos-move/aptos-aggregator/src/resolver.rs

```diff
@@ -82,9 +82,7 @@ pub trait TAggregatorView {
 
     /// Returns a unique per-block identifier that can be used when creating a
     /// new aggregator V2.
-    fn generate_aggregator_v2_id(&self) -> Self::IdentifierV2 {
-        unimplemented!("ID generation for Aggregator V2 is not yet supported")
-    }
+    fn generate_aggregator_v2_id(&self) -> Self::IdentifierV2;
```

Comment:
Just trying to learn. What's the difference between this and using "unimplemented" as before?

Reply:
Before we had a default implementation (or better to say no implementation), now we require one